modules/social_features/social_branding/src/Plugin/GraphQL/SchemaExtension/AppSchemaExtension.php

```diff
@@ -138,6 +141,12 @@ public function registerResolvers(ResolverRegistryInterface $registry) {
       $builder->produce('color_alpha')
         ->map('color', $builder->fromParent())
     );
+
+    // PlatformFeature fields.
+    $registry->addFieldResolver('PlatformFeature', 'machineName',
+      $builder->produce('platform_feature_machine_name')
```

Comment:
I used a custom producer. Is not there a data producer to get a key from a keyed array?

Reply:
There is the `seek` operator for numeric arrays, but that doesn't quite do what you're looking for here. However, see also my comment about creating a class to contain the data rather than an array. :)